### PR TITLE
Update CLI options, QoS description

### PR DESCRIPTION
**Pull request by gilgongo**
# Does this need translation?

- [x] Yes 
- [ ] No 

# Changes

Re-format into sections like -h output. Add missing options. Update QoS description as agreed in https://github.com/jamulussoftware/jamuluswebsite/pull/421#issuecomment-817363424

### CODE REVIEW

File: wiki/en/en-Command-Line-Options.md [hunk 1]
```diff
@@ -7,51 +7,63 @@ permalink: "/wiki/Command-Line-Options"
...
+| `-h,-?` | `--help`          | This help text                                                                                               |                                                          |
+| `-i`    | `--inifile`       | Set location of initialization file (overrides default)                                                      | Client (always) and server (with GUI)                    |
+| `-n`    | `--nogui`         | Disable GUI                                                                                                  |                                                          |
+| `-p`    | `--port`          | Local UDP port number. Default is 22124                                                                      |                                                          |
+|    `-Q` | `--qos`        | Sets the quality of service DSCP value. Default is 128. Disable with 0.  | QoS is ignored by Windows, but see [Tips & Tricks](Tips-Tricks-More#quality-of-service) for how to apply it. |
```
Comment (pljones) [inline wiki/en/en-Command-Line-Options.md]: Alignment.

Comment (pljones) [reply wiki/en/en-Command-Line-Options.md:34]: And I thought this was General, not Client?

File: wiki/en/en-Tips-Tricks-More.md [hunk 2]
```diff
@@ -110,20 +110,12 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
...
 
-Prior to Windows Vista / Server 2008, Windows allowed software developers to set the
-Quality of Service field on IP traffic.
-With the release of Vista / Server 2008, Microsoft began to overwrite DSCP values
-that were set by applications with the Policy-based QoS layer.
+Jamulus’s QoS settings (including the default) have no effect on Windows because the operating system ignores them. To enable Quality of Service for Jamulus, you must follow these instructions. Note also that you may need to repeat this procedure every time Jamulus is updated.
 
-In order to enable Quality of Service you must follow these instructions:
 
 In Search box beside Start menu Type: Local Group Policy Editor (enter)<br>
```
Comment (pljones) [inline wiki/en/en-Tips-Tricks-More.md:120]: Should these points be a bulleted or numbered list?

Comment (gilgongo) [reply wiki/en/en-Tips-Tricks-More.md:120]: I don't know what the significance of the indents is, so I don't know what the effect of converting them to a numbered or bulleted list would be. @DavidSavinkoff do you know? BTW I think we can remove the \<br\> tags.

**Comment by ghost** [comment]
The indenting is similar to a 'tree' listing of a directory. This is a representation of what you see while doing the 'Windows procedure' on Windows. If you have Windows, you should attempt to follow the directions I gave so that you see what I'm trying to show here. Never mind the indentation, maybe my directions require interpretation! I was thinking that setting DCSP for Jamulus could be rolled into the installation package for Windows (I'm guessing this makes sense).

**Review by ghost — COMMENTED** [review]
Line 38: wiki/en/en-Command-line-Options.md
`### Sever only`
should be:
`### Server only`

**Comment by gilgongo** [comment]
> I was thinking that setting DCSP for Jamulus could be rolled into the installation package for Windows

That would certainly be good.

File: wiki/en/en-Command-Line-Options.md [hunk 3]
```diff
@@ -7,51 +7,63 @@ permalink: "/wiki/Command-Line-Options"
...
-"C:\Program Files (x86)\Jamulus\Jamulus.exe" -s -R "C:\path\to\your\Jam Recordings"
+"C:\Program Files (x86)\Jamulus\Jamulus.exe" --inifile "C:\path\to\myinifile.ini"
 ```
 
 For macOS, start a Terminal window and run Jamulus with the desired options like this:
 
 ```shell
-/Applications/Jamulus.app/Contents/MacOS/Jamulus -s -R "/path/to/your/jam recordings"
+/Applications/Jamulus.app/Contents/MacOS/Jamulus --inifile "/path/to/myinifile.ini"
 ```
```
Comment (jujudusud) [inline wiki/en/en-Command-Line-Options.md:22]: Linux ?
Do we have to say something about Linux ? or only talking about Windows and mac ?

File: wiki/en/en-Command-Line-Options.md [hunk 4]
```diff
@@ -7,51 +7,63 @@ permalink: "/wiki/Command-Line-Options"
...
-|    `-s` |`--server`         | Start in server mode | (server only) [See note](Choosing-a-Server-Type) |
-|    `-t` |`--notranslation`  | Disable translations | (client and server) |
-|    `-T` |`--multithreading` | Enable multithreading | (server only) |
-|    `-u` |`--numchannels`    | Maximum number of users. Default is 10, maximum is 150 | (server only) |
-|    `-w` |`--welcomemessage` | Supports HTML and inline CSS formatting (in enclosing quotes), or set path to text file. | (server only) |
-|    `-z` |`--startminimized` | Start minimized | (server only) |
-|       |`--ctrlmidich`     | MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels). Format: `channel[;f<off>*<channels>][;p<off>*<channels>][;s<off>*<channels>][;m<off>*<channels>]` | (client only) see [Tips & Tricks](Tips-Tricks-More#Using-ctrlmidich-for-MIDI-controllers) |
-|       |`--clientname`     | Window title and JACK client name | (client only) |
-|       |`--serverpublicip` | Supply the server's public IP address  | (server only) [See note](Choosing-a-Server-Type) |
+_Note that the following options will not alter settings made in the GUI._
```
Comment (jujudusud) [inline wiki/en/en-Command-Line-Options.md:24]: I think, the text on line 24 is not visible and will be ignored by readers.

**Pull request merged**